gateway/pom.xml

```diff
@@ -110,6 +119,17 @@
       <artifactId>ldaptive-netscape</artifactId>
       <version>1.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20230618</version>
+    </dependency>
+    <dependency>
+      <groupId>javax</groupId>
```

Comment:
Can you add a comment on why this is needed? otherwise it'll look strange on a webflux project